numpy/core/src/umath/ufunc_object.c

```diff
@@ -4457,7 +4470,8 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc, PyObject *args,
     if (operation == UFUNC_REDUCEAT) {
         PyArray_Descr *indtype;
         indtype = PyArray_DescrFromType(NPY_INTP);
-        if (!PyArg_ParseTupleAndKeywords(args, kwds, "OO|OO&O&:reduceat", reduceat_kwlist,
+        if (!PyArg_ParseTupleAndKeywords(args, kwds, "OO|OO&O&:reduceat", 
+                                         reduceat_kwlist,
```

Comment:
It's good practice not to reformat lines unrelated to your change.

Reply:
I think I might have hit a key while I was searching around my bad.